fava/static/sass/_base.scss

```diff
@@ -595,3 +630,36 @@ table.queryresults {
         margin-left: 3px;
     }
 }
+
+/* Modifications for ipad */
+@media (min-width: 768px) {
+    body {
+        padding: $header_height 0 0 $aside_width;
+    }
+
+    aside {
+        margin-left: 0;
+    }
+    #aside_button {
+        display: none;
+    }
+}
+
+/* At twice min-width + aside_width, it's safe to make two columns */
+/* If min_width is 560px, $safe_two_column is 1280px */
+@media (min-width: $safe_two_column) {
```

Comment:
I'd prefer to see the desktop CSS code be the default and the changes for small devices scoped instead. It's not like small phones are our main target...